samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -295,7 +293,7 @@ static void audio_timer_timeout(struct k_work *work)
 	 * data going to the server)
 	 */
 	for (size_t i = 0U; i < configured_sink_stream_count; i++) {
-		struct bt_audio_stream *stream = &streams[i];
+		struct bt_audio_stream *stream = &sinks[i].stream;
```

Comment:
I agree, maybe it would be better to name it `tx` and `rx` respectively?

Reply:
> I agree, maybe it would be better to name it `tx` and `rx` respectively?

I think so :) 

Or c_to_s (client to server) and s_to_c if you want to use an absolute reference, instead of relative, but TX/RX is probably best